pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -13,6 +13,7 @@ in
 , noSysDirs
 , perl
 , zlib
+, runCommandNoCC
```

Comment:
This needs to be `runCommand`; `runCommandNoCC` is an alias to it that cannot be used in Nixpkgs.

Reply:
Done. Also retargeting now.